### ISSUE TITLE
Command Palette (a.k.a. Quick Actions)

### ISSUE THREAD

**Issue by vladstudio** [penpot/penpot#1107]
Command Palettes (or Quick Actions) are a super-useful part of applications such as PenPot! They allow you to:
1) discover functionality
2) access commands quickly.

It'd be awesome to have Comand Palette in Penpot. I realize it's no easy task but maybe you will find it worth it! Examples of Command Palettes in various programs:

Alfred: 
![image](https://user-images.githubusercontent.com/914320/126648428-892eb5b3-f8a0-4d2f-964c-96a1c5cc5731.png)

Vivaldi:
![image](https://user-images.githubusercontent.com/914320/126648517-fda3408d-5185-4600-a1cf-d29f4fc6ad75.png)

VSCode:
![image](https://user-images.githubusercontent.com/914320/126648734-da1555f8-1f2a-4b03-89c7-a70e69af1a2e.png)

and of course Figma:
![image](https://user-images.githubusercontent.com/914320/126648698-1d57f76f-7b0d-4108-bf9c-4301fa7d9e6d.png)

**Comment by elhombretecla** [comment]
Good point. Command palettes are really useful and increase the workflow speed.

We need to validate it and check how to implement it.

Thank you for your feedback!

[https://tree.taiga.io/project/penpot/issue/1905](https://tree.taiga.io/project/penpot/issue/1905)

**Comment by einarpersson** [comment]
I just tried penpot and this was the first thing I was looking for.  I dont want to invest time in reading docs / tutorials and learning a bunch of shortcuts, just give me a command palette with some fuzzy search and I can discover what I need by myself. 

Note: The palette should show the shortcut (if configured) matching the command, that way you will learn the shortcuts by yourself for the commands that you use often.

As a casual user (as in, not working in Figma/Penpot everyday) this is so important that it may actually be a cause enough to not make the switch from Figma (yet). Discoverability is key!

**Comment by salernoelia** [comment]
Any news on this? Would be a life changing feature.

**Comment by revolter** [comment]
And it must have <kbd>cmd</kbd> + <kbd>k</kbd> as the keyboard shortcut, as it's already somewhat of a standard (e.g. Notion).
